src/ui/common/src/components/workflows/WorkflowSettings.tsx

```diff
@@ -499,7 +502,11 @@ const WorkflowSettings: React.FC<WorkflowSettingsProps> = ({
           display="inline"
         >
           Update Mode:{' '}
-          {sortedObjects.map((object) => `${object.update_mode}`).join(', ')}
+          {sortedObjects
+            .map(
+              (object) => `${object.spec.parameters.update_mode || 'replace'}`
```

Comment:
The 'replace' is currently hard coded because S3LoadParams don't have an `update_mode` field.

Reply:
nit: Can we make a const enum for the update modes that we have. UPDATE_MODE.REPLACE would be nice to be able to use here.